gradle/dependencies.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
         exclude group: 'org.ow2.asm'
     }
 
+    implementation 'com.android.tools:r8:2.1.16-dev'
```

Comment:
Wouldn't you want a non-dev build?

Reply:
yes, this is mainly a proof-of-concept, I did a lot of testing lately with different versions of r8, maybe the latest release 1.6.84 already includes all things that I need, I will check.

Reply:
Right now we only release non-dev builds to Google Maven when a stable release of Android Studio is released. Which is why there is no 2.0 or 2.1 versions there yet.

Reply:
I checked, the main reason to use a 2.x release is the JdkClassFileProvider. It is needed to support JDK 9+.

Reply:
changed to the latest non-dev release 2.0.61.